corehq/apps/reports/static/reports/ko/saved_reports.js

```diff
@@ -326,6 +326,6 @@ $.fn.reportUserConfigurableConfigEditor = function (options) {
 $.fn.reportConfigList = function (options) {
     this.each(function(i, v) {
         var viewModel = new ReportConfigsViewModel(options);
-        ko.applyBindings(viewModel, $(this).get(i));
+        $(this).koApplyBindings(viewModel);
```

Comment:
I don't think this is the same. Looks like the previous one was trying to iterate through a list of nodes and bind each of them. (Which... it'd be better if the looping was done in the template, but that's not on you of course.) I think if you use http://api.jquery.com/eq/ it'll work:

```javascript
$(this).eq(i).koApplyBindings(viewModel)
```

but you probably want to test that out, or at least verify that that function exists in the version of jquery that we're using.